core/sector_storage/impl/remote_worker.cpp

```diff
@@ -60,7 +60,8 @@ namespace fc::sector_storage {
 
   outcome::result<std::set<primitives::TaskType>>
   RemoteWorker::getSupportedTask() {
-    return api_.TaskTypes();
+    OUTCOME_TRY(supported_types, api_.TaskTypes());
+    return std::move(supported_types);
```

Comment:
no, it doesn't because it's not `auto supported_types`, but `auto &supported_types{_outcome_supported_types}` and compiler gives warning asking to add explicit `std::move`